ReactiveCocoa/Swift/Property.swift

```diff
@@ -355,61 +325,61 @@ public struct AnyProperty<Value>: PropertyType {
 	}
 
 	/// Initializes a property as a read-only view of the given property.
-	public init<P: PropertyType where P.Value == Value>(_ property: P) {
-		sources = AnyProperty.capture(property)
+	public init<P: PropertyProtocol where P.Value == Value>(_ property: P) {
+		sources = AnyProperty.capturing(property)
 		disposable = nil
 		_value = { property.value }
 		_producer = { property.producer }
 		_signal = { property.signal }
 	}
 
-	/// Initializes a property that first takes on `initialValue`, then each value
+	/// Initializes a property that first takes on `initial`, then each value
 	/// sent on a signal created by `producer`.
-	public init(initialValue: Value, producer: SignalProducer<Value, NoError>) {
-		self.init(propertyProducer: producer.prefix(value: initialValue),
+	public init(initial: Value, followingBy producer: SignalProducer<Value, NoError>) {
+		self.init(sourcingFrom: producer.prefix(value: initial),
 		          capturing: [])
 	}
 
-	/// Initializes a property that first takes on `initialValue`, then each value
+	/// Initializes a property that first takes on `initial`, then each value
 	/// sent on `signal`.
-	public init(initialValue: Value, signal: Signal<Value, NoError>) {
-		self.init(propertyProducer: SignalProducer(signal: signal).prefix(value: initialValue),
+	public init(initial: Value, followingBy signal: Signal<Value, NoError>) {
+		self.init(sourcingFrom: SignalProducer(signal: signal).prefix(value: initial),
 		          capturing: [])
 	}
 
 	/// Initializes a property by applying the unary `SignalProducer` transform on
 	/// `property`. The resulting property captures `property`.
-	private init<P: PropertyType>(_ property: P, @noescape transform: SignalProducer<P.Value, NoError> -> SignalProducer<Value, NoError>) {
-		self.init(propertyProducer: transform(property.producer),
-		          capturing: AnyProperty.capture(property))
+	private init<P: PropertyProtocol>(sourcingFrom property: P, transform: @noescape (SignalProducer<P.Value, NoError>) -> SignalProducer<Value, NoError>) {
```

Comment:
I think we should leave `property` as an unnamed argument. That fits the general RAC style of taking `Signal`s and `SignalProducer`s as unnamed arguments.